AssetManagementTestProject/PageObj/LeftMenuPage.cs

```diff
@@ -3,26 +3,25 @@
 namespace AssetManagementTestProject.PageObj;
 public class LeftMenuPage : WebDriverAction
 {
-    private readonly string btnHomeInMenu = "//a[text() = 'Home']";
-    private readonly string btnManageUserInMenu = "//a[text() = 'Manage User']";
-    private readonly string btnManageAssetInMenu = "//a[text() = 'Manage Asset']";
-    private readonly string btnManageAssignmentInMenu = "//a[text() = 'Manage Assignment']";
-    private readonly string btnManageReturningInMenu = "//a[text() = 'Manage Returning']";
-    private readonly string btnReportInMenu = "//a[text() = 'Report']";
-
+    public readonly string BtnHomeInMenu = "//a[text() = 'Home']";
+    public readonly string BtnManageAssetInMenu = "//a[text() = 'Manage Asset']";
+    public readonly string BtnManageAssignmentInMenu = "//a[text() = 'Manage Assignment']";
+    public readonly string BtnManageReturningInMenu = "//a[text() = 'Manage Returning']";
+    public readonly string BtnManageUserInMenu = "//a[text() = 'Manage User']";
+    public readonly string BtnReportInMenu = "//a[text() = 'Report']";
     public LeftMenuPage() : base()
     {
 
     }
     public List<string> ReturnMenuBar()
     {
         List<string> menuBar = new List<string>();
-        menuBar.Add(btnHomeInMenu);
-        menuBar.Add(btnManageUserInMenu);
-        menuBar.Add(btnManageAssetInMenu);
-        menuBar.Add(btnManageAssignmentInMenu);
-        menuBar.Add(btnManageReturningInMenu);
-        menuBar.Add(btnReportInMenu);
+        menuBar.Add(BtnHomeInMenu);
```

Comment:
instead of adding a list of elemenets, you can return each element to the test case for clearer verification

Reply:
I fixed it. Thank you